File: official/guides/getting-started/csharp/create_to_address.cs
Dictionary<string, object> addressData = new Dictionary<string, object>() {
  {"name", "George Costanza"},
  {"company", "Vandelay Industries"}
  {"street1", "1 E 161st St."},
  {"city", "Bronx"},
  {"state", "NY"},
  {"zip", "10451"}
};

Address toAddress = await Address.Create(addressData);

File: official/guides/getting-started/node/create_to_address.js
const toAddress = new api.Address({
    name: 'George Costanza',
    company: 'Vandelay Industries',
    street1: '1 E 161st St.',
    city: 'Bronx',
    state: 'NY',
    zip: '10451',
});

toAddress.save().then(console.log);
